### PR TITLE
Fixed Broken Link and Typo

### PR DESCRIPTION
**Pull request by ansijain**
Signed-off-by: ansijain <ansi.jain@india.nec.com>

### CODE REVIEW

**Comment by mazzystr** [comment]
/approve
/lgtm

Thanks for the fixes!

**Comment by ansijain** [comment]
Fixed one more link 
@mazzystr can you please have a look

File: labs/lab000/lab000.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The clusters for the lab are deployed externally, and the only requisite to foll
 More information about KubeVirt and related components can be found here:
 
 - [KubeVirt](http://kubevirt.io/)
-- [KubeVirt user guide](https://kubevirt.io/user-guide/docs/latest/welcome/index.html)
+- [KubeVirt user guide](https://kubevirt.io/user-guide/welcome/)
```
Comment (mazzystr) [inline labs/lab000/lab000.md:31]: Can you change to simply `https://kubevirt.io/user-guide`? [user-guide/pull/425](https://github.com/kubevirt/user-guide/pull/425) is going to remove the `welcome` uri

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Comment by mazzystr** [comment]
Thanks!

/approve
/lgtm

**Review by mazzystr — APPROVED** [review]

**Comment by ansijain** [comment]
@mazzystr Can you please have a look and let me know what should I do
as test is failing and giving error ERROR: This script does not work on Python 3.5 The minimum supported Python version is 3.6. Please use https://bootstrap.pypa.io/pip/3.5/get-pip.py instead.

**Pull request merged**